backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Altinn.AccessManagement.UI.Mocks.csproj

```diff
@@ -12,6 +12,18 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I recently found out why the IDE still adds all test data files individually to the project file.
If you change the catch'all CopyToOutputDirectory rule above to:
```
  <ItemGroup>
    <None Include="Data\**">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
  </ItemGroup>
```
This shouldn't happen anymore and you can remove these individual file references

Reply:
Nice! I'll do that - thanks! 😄